src/main/java/space/obminyashka/items_exchange/rest/LocationController.java

```diff
@@ -173,5 +172,53 @@ public ResponseEntity<String> createLocationsInitFile(@RequestBody LocationsRequ
         } catch (IOException e) {
             return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(e.getMessage());
         }
+
+    }
+    @PreAuthorize(HAS_ROLE_ADMIN)
```

Comment:
Incorrect formatting (excess new line in 175 line and no new line after 177 line)

Reply:
done